recipes/epigen/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "epigen" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/biomedbigdata/epigen/archive/refs/heads/master.zip
```

Comment:
the URL should point to a specific tag or release, ([ref](https://bioconda.github.io/contributor/guidelines.html#stable-url)). otherwise this recipe will break if there are ever any new commits to the master branch. it looks like this repo hasn't been updated in a while, but better to be on the safe side. you can also point to a specific commit [by its hash](https://docs.github.com/en/repositories/working-with-files/using-files/downloading-source-code-archives#source-code-archive-urls)

Reply:
thank you @kbseah. I will update the recipe to point to a specific commit as you recommended.